draft-ietf-webpush-protocol.xml

```diff
@@ -938,6 +938,11 @@ Host: push.example.net
           then the push service MUST return a 410 (Gone) response to the application server
           monitoring the receipt subscription.
         </t>
+        <t><list style="hanging"><t hangText="Note:">
+          An application server also receives the push message resource URI when it sends
+          a push message.  This allows an application server to acknowledge messages it

```

Comment:
The PS should probably send an ACK anyway.  Otherwise it needs special logic to identify who made the request.